.gitignore

```diff
@@ -6,3 +6,4 @@
 *.dll
 _build
 .idea
+/froi/algorithm/qimageview.pyd
```

Comment:
I think there is no need to add this line as we already have a '*.pyd' expression.